### ISSUE TITLE
Configure via package.json?

### ISSUE THREAD

**Issue by ianwremmel** [zemirco/bannerify#1]
I'm not sure this is even possible with browserify right now, but is it possible to configure this via package.json?

I'm maintaining a fairly extensive library that needs a banner at the header of every source file for legal reasons, but once the library is browserified, I don't see any need to repeat the banner. Instead, I'd like to configure bannerify in package.json (just like package-specific transforms) to just add the banner once.

**Comment by zemirco** [comment]
You can configure `bannerify` via your `package.json`. The banner is only added once for your whole bundle. Just put your license into your `package.json` and use it your template.

`package.json`

```
...
"license": "MIT (or some more text)",
...
```

`banner.txt`

```
/**
 * @license <%= pkg.license %>
 */
```

Does that help?

**Comment by ianwremmel** [comment]
Not exactly. The part I'm having trouble figuring out is how to get it to run when another project requires the library. For this library, the package.json already includes

```
Browserify: {
  Transform: [
    Envify,
    Uglifyify
  ]
}
```

to guarantee that any project the `require`s it includes those transforms (sorry for the syntactic errors there, typing json in an iPhone is kinda tricky). I'd like to be able to do something similar to guarantee a banner gets included once and only once when dependent projects bundle this library.
